cloudbuild.yaml

```diff
@@ -144,7 +145,40 @@ steps:
         /workspace/bin/oscoin-cli
       )
 
-      echo "$${bins[*]}" | xargs -P8 -n1 ./scripts/docker/build-image.sh
+      echo "$${bins[*]}" | xargs -P8 -n1 ./scripts/docker/publish-image.sh
+
+  - id: Deploy
+    waitFor:
+    - Publish images
+    name: "gcr.io/cloud-builders/gcloud"
+    entrypoint: bash
+    env:
+    - BRANCH_NAME=$BRANCH_NAME
+    - OSCOIN_VERSION=$COMMIT_SHA
+    - OSCOIN_NETWORK=devnet
+    - OSCOIN_BENEFICIARY=yrbonyyyoeyfoebik1y6pmradnkyjakqj9s14ckf4kgwoetoskd9apqwan7wwgrfmweoydesdero
```

Comment:
Funny how this address has the word `desde` at the end, which apparently it's a valid Spanish word 😀